src/io/flutter/inspector/JumpToSourceAction.java

```diff
@@ -8,8 +8,13 @@
 import com.intellij.xdebugger.XSourcePosition;
 import com.intellij.xdebugger.frame.XNavigatable;
 import com.intellij.xdebugger.frame.XValue;
+import org.jetbrains.annotations.NotNull;
 
 public class JumpToSourceAction extends JumpToSourceActionBase {
+  public JumpToSourceAction() {
+    super(InspectorActions.JUMP_TO_SOURCE);
```

Comment:
@devoncarew : what's the preferred format for these IDs?  Would "jumpToSource" be more consistent?

Reply:
FWIW, updated...